package/kernel/linux/modules/block.mk

```diff
@@ -26,7 +26,9 @@ define KernelPackage/ata-core
   SUBMENU:=$(BLOCK_MENU)
   TITLE:=Serial and Parallel ATA support
   DEPENDS:=@PCI_SUPPORT||TARGET_sunxi +kmod-scsi-core
-  KCONFIG:=CONFIG_ATA
+  KCONFIG:= \
+	CONFIG_SATA_PMP=y \
```

Comment:
Since buildbots build with all kmods enabled and this is =y, wouldn't this increase the kernel size for _all_ targets?